src/System.Private.CoreLib/shared/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -44,18 +44,25 @@ private enum InternalState
         // Contains the reference to VM's representation of the AssemblyLoadContext
         private readonly IntPtr _nativeAssemblyLoadContext;
 
-        protected AssemblyLoadContext() : this(false, false)
+        protected AssemblyLoadContext() : this(false, false, null)
         {
         }
 
-        protected AssemblyLoadContext(bool isCollectible) : this(false, isCollectible)
+        protected AssemblyLoadContext(bool isCollectible) : this(false, isCollectible, null)
         {
         }
 
-        private protected AssemblyLoadContext(bool representsTPALoadContext, bool isCollectible)
+        public AssemblyLoadContext(string name, bool isCollectible = true) : this(false, isCollectible, name)
```

Comment:
isCollectible is also supposed to default to false

Reply:
I assume these are notes for you to fix these, right? :-)

Reply:
Fixed in #23506